sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -9,6 +9,17 @@ import { logger } from "../util/logging";
 
 import * as child_process from "child_process";
 
+function get_safe_working_dir(): string {
+    if (process.platform == "win32") {
+      if (!process.env.SystemRoot) {
+        throw new Error("Azure CLI credential expects a 'SystemRoot' environment variable");
+      }
+      return process.env.SystemRoot;
+    } else {
+      return "/bin";
```

Comment:
any reason we always assume bin on non-Windows and don't consult the variable?

Reply:
What variable would we consult outside of Windows?  @schaabs had mentioned defaulting to /bin because (I'm guessing) it's the one defined in POSIX to be there.

Reply:
@xirzec I'd be happy to change this to an environment variable, I just wasn't aware of one which was available across all distros. Also, as this is a preview the implementation is subject to change, so I didn't spend a huge amount of time investigating. It's likely that we'll end up resolving the path to the Azure CLI manually in our code in which case we'd set the working directory to be the CLI install directory.

Reply:
Looking through the environment variables on my Ubuntu box, I don't see any pointing to /bin that we could use. I'm happy to hardwire it for now and then change later as @schaabs suggests.